### ISSUE TITLE
[PWA widgets] Add documentation about options to update widgets more than once a day

### ISSUE THREAD

**Issue by davesmits** [MicrosoftDocs/edge-developer#3018]
There are no noted restrictions about the update time. It kinda suggest the update time can be anything but everything below the values used in the doc (86.400) wont work

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 1a0e6a65-5e0a-55cc-d3c8-b4fdd37a2d2b
* Version Independent ID: 1a0e6a65-5e0a-55cc-d3c8-b4fdd37a2d2b
* Content: [Build PWA-driven widgets - Microsoft Edge Development](https://learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/widgets)
* Content Source: [microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md](https://github.com/MicrosoftDocs/edge-developer/blob/main/microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md)
* Product: **microsoft-edge**
* Technology: **pwa**
* GitHub Login: @MSEdgeTeam
* Microsoft Alias: **msedgedevrel**

AB#48547685

**Comment by captainbrosset** [comment]
Thanks a lot for this issue. 
Can you say a little more about what happens when you choose a value that's less than 86400? What doesn't work exactly? Does the widget not get registered? Something else?

**Comment by captainbrosset** [comment]
Fair point. I'm going to discuss about this with the engineering team who knows best about service workers and widgets and we'll get back to you.

**Comment by captainbrosset** [comment]
> Push Notification are required to use showNotification resuling in a toast banner

Correct, silent push notifications are currently not supported by browsers, at least by Chrome and Edge.

> widget resume seems to work pretty nicely. Thanks!

Oh great! Glad this is working for you. I'll add this issue to our backlog and use it to update the docs to mention how to update widgets more than once a day.
